control/statefbk.py

```diff
@@ -389,6 +390,12 @@ def lqr(*args, **keywords):
     --------
     lqe
 
+    Notes
+    -----
+    The return type for `K` and `S` depends on the default class set for
+    state space operations.  By default, this is the Numpy `matrix`
+    class in this release, but this can be reconfigured using the
+    :func:`~control.use_numpy_matrix` function.
```

Comment:
This is also true for most of the other functions in this file. Maybe a general note referenced here and in those functions would be preferable. Cf. #439